index.markdown

```diff
@@ -0,0 +1,51 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: default
+---
+# Enforcing, Inferring, and Synthesizing Optional Properties (EISOP) Framework
+
+The Enforcing, Inferring, and Synthesizing Optional Properties (EISOP) Framework project
+aims to provide a unified framework for optional properties in Java.
+
+Development occurs in the
+[Checker Framework eisop fork](https://github.com/eisop/checker-framework).
+The focus in this fork is on soundness and usability.
+One (closed-source) user of the EISOP Dataflow Framework reported a 10 percent
+reduction in false positives compared to the typetools Dataflow Framework.
+
+Releases can be downloaded from
+[GitHub](https://github.com/eisop/checker-framework/releases) and
+[Maven Central](https://central.sonatype.dev/namespace/io.github.eisop).
+
+More research results can be found on a
+[separate page](https://opprop.github.io/).
+
+The [javac diagnostics wrapper](https://github.com/eisopux/javac-diagnostics-wrapper)
+is a wrapper around javac to output diagnostics in an easily-configurable way.
+
+[Checker Framework Manual](/cf/index.md)
```

Comment:
This link should be earlier, with the paragraph `Development occurs in the`.
Is the `index.md` part needed?